### DISCUSSION TITLE
ccos: A CentOS-based CoreOS-style bootc image

### DISCUSSION THREAD

**Discussion by castrojo** [bootc-dev/bootc#1000] [General]
@bsherman has started work on a "What would a CoreOS inspired image look like, built on CentOS?" 

Base image work is here, enjoy:

https://github.com/ublue-os/ccos

**Comment by travier** [comment]
Note that we maintain manifests to build CentOS Stream CoreOS (SCOS) for OKD as part of https://github.com/openshift/os. The images are not published outside of OKD but it should be possible to build and publish them.
